snapcraft.yaml

```diff
@@ -555,6 +555,30 @@ parts:
       cp -r usr/share/locale-langpack $CRAFT_PART_INSTALL/usr/share/
 
 
+  poppler:
+    after: [ cairo, gdk-pixbuf, glib, gobject-introspection, gtk3, meson-deps ]
+    source: https://gitlab.freedesktop.org/poppler/poppler.git
+    source-tag: 'poppler-22.07.0'
+    source-depth: 1
+    plugin: cmake
+    cmake-parameters:
+      - -DCMAKE_INSTALL_PREFIX=/usr
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_GTK_TESTS=OFF
+      - -DBUILD_QT5_TESTS=OFF
+      - -DBUILD_QT6_TESTS=OFF
+      - -DBUILD_CPP_TESTS=OFF
+      - -DBUILD_MANUAL_TESTS=OFF
+    build-environment: *buildenv
+    build-packages:
+      - libboost1.74-dev
+      - libcurl4-openssl-dev
+      - libfreetype6-dev
+      - liblcms2-dev
+      - libopenjp2-7-dev
+      - libnss3-dev
```

Comment:
I prefer to list the dependencies that are required rather than relying upon another part to provide the packages as that ensures that changes to the other part do not cause knock-on effects. However, I've made the change as this isn't my project to enforce my stylistic choices upon :-).